projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -70,7 +75,33 @@ def __init__(self, config_file=None):
         )
 
         # The Control Service API URL (http://url/data-jobs) without data-jobs suffix
-        self.control_api_url = self.get_value("CONTROL_API_URL")
+        self.control_api_url = self._get_atleast_one_value(
+            "CONTROL_API_URL", "VDK_CONTROL_SERVICE_URL"
+        )
+
+        # Set the VDK properties API URL
+        self.properties_api_url = self.get_value(
+            "VDK_PROPERTIES_URL", is_required=False
+        )
+        if self.properties_api_url:
+            self.set_value("PROPERTIES_API_URL", self.properties_api_url)
+
+        # Set the VDK properties API authorization URL
+        self.properties_api_authorization_url = self.get_value(
+            "VDK_PROPERTIES_TOKEN_AUTHORIZATION_URL", is_required=False
+        )
+        if self.properties_api_authorization_url:
+            self.set_value(
+                "PROPERTIES_API_TOKEN_AUTHORIZATION_URL",
+                self.properties_api_authorization_url,
+            )
+
+        # Set the VDK properties API token
+        self.properties_api_token = self.get_value(
+            "VDK_PROPERTIES_TOKEN", is_required=False
+        )
+        if self.properties_api_token:
+            self.set_value("PROPERTIES_API_TOKEN", self.properties_api_token)
```

Comment:
Those are not necessary to be set by vdk heartbeat.

Reply:
Removed, thanks for the review